projects/plugins/jetpack/modules/stats.php

```diff
@@ -461,6 +549,21 @@ function stats_reports_page( $main_chart_only = false ) {
 			?>
 			</h2>
 		</div>
+		<div class="wrap">
+			<div class="stats-odyssey-notice">
+				<div class="stats-odyssey-notice--content">
+					<h2 class="stats-odyssey-notice--content-header"><?php esc_html_e( 'Old Jetpack Stats experience', 'jetpack' ); ?></h2>
+					<p class="stats-odyssey-notice--content-text"><?php esc_html_e( 'The old Jetpack Stats has been deprecated and will be removed soon.', 'jetpack' ); ?></p>
+					<div class="stats-odyssey-notice--action-bar">
+						<button class="dops-button stats-odyssey-notice--primary-button">
```

Comment:
The button enables the new experience and doesn't require users to visit the settings page.